### DISCUSSION TITLE
Fontawesome not properly loaded

### DISCUSSION THREAD

**Discussion by adamgayoso** [pradyunsg/furo#400] [Potential Issue] [answered]
Hello,

I followed the instructions to add fontawesome icons to the footer, but end up getting this. On another note, fontawesome with sphinx design is [not working with furo](https://github.com/executablebooks/sphinx-design/issues/38)

<img width="101" alt="Screen Shot 2022-03-15 at 8 03 48 AM" src="https://user-images.githubusercontent.com/10859440/158407608-dd2bceba-fdea-4d60-90e1-324d08791b9c.png">

**Comment by pradyunsg** [comment]
Hiya!

There isn't enough information here for me to be able to help. I don't even have a way to know what you've tried or what the failure page's contents are. :)

As a cursory question, did you set `html_css_files`?

**Comment by umarcor** [comment, marked as answer]
See https://github.com/pradyunsg/furo/discussions/590 and https://github.com/pradyunsg/furo/pull/629.
